content/guides/asset-management.md

```diff
@@ -4,163 +4,522 @@ sort: 3
 ```
 
-Now, when you `import Image from './my-image.png'`, that image will be processed and added to your `output` directory _and_ the `Image` variable will contain the final url of that image after processing. When using the [css-loader](/loaders/css-loader), as shown above, a similar process will occur for `url('./my-image.png')` within your CSS. The loader will recognize this is a local file, and replace the `'./my-image.png'` path with the final path to the image in your `output` directory. The [html-loader](/loaders/html-loader) handles `<img src="./my-image.png" />` in the same manner.
+__src/style.css__
 
-T> The next step is minifying and optimizing your images. Check out the [image-webpack-loader](https://github.com/tcoopman/image-webpack-loader) and [url-loader](/loaders/url-loader) for more on how you can enhance your image loading process.
+``` diff
+.hello {
+  color: red;
++   background: url('./icon.png');
+}
+```
+
+Let's create a new build and open up the index.html file again:
+
+``` bash
+npm run build
+
+Hash: 854865050ea3c1c7f237
+Version: webpack 2.6.1
+Time: 895ms
+                               Asset     Size  Chunks                    Chunk Names
+5c999da72346a995e7e2718865d019c8.png  11.3 kB          [emitted]
+                           bundle.js   561 kB       0  [emitted]  [big]  main
+   [0] ./src/icon.png 82 bytes {0} [built]
+   [1] ./~/lodash/lodash.js 540 kB {0} [built]
+   [2] ./src/style.css 1 kB {0} [built]
+   [3] ./~/css-loader!./src/style.css 242 bytes {0} [built]
+   [4] ./~/css-loader/lib/css-base.js 2.26 kB {0} [built]
+   [5] ./~/style-loader/lib/addStyles.js 8.7 kB {0} [built]
+   [6] ./~/style-loader/lib/urls.js 3.01 kB {0} [built]
+   [7] (webpack)/buildin/global.js 509 bytes {0} [built]
+   [8] (webpack)/buildin/module.js 517 bytes {0} [built]
+   [9] ./src/index.js 503 bytes {0} [built]
+```
+
+If all went well, you should now see your icon as a repeating background, as well as an `img` element behind our `Hello webpack` text. If you inspect this element, you'll see that actual filename has changed to something like `5c999da72346a995e7e2718865d019c8.png`. This means webpack found our file in the `src` folder and processed it!
+
+T> A logical next step from here is minifying and optimizing your images. Check out the [image-webpack-loader](https://github.com/tcoopman/image-webpack-loader) and [url-loader](/loaders/url-loader) for more on how you can enhance your image loading process.
 
 
 ## Loading Fonts
 
-So what about other assets like fonts? The file and url loaders will take any file you load through them and output it to your build directory. This means we can use them for any kind of file, including fonts:
+So what about other assets like fonts? The file and url loaders will take any file you load through them and output it to your build directory. This means we can use them for any kind of file, including fonts. Let's adjust our `webpack.config.js` file to instruct it to handle font files:
```

Comment:
I would simplify:

``Let's adjust our `webpack.config.js` file to instruct it to handle font files:``

to

``Let's update our `webpack.config.js` to handle font files:``